configs/reconnaissance_demo/common.conf

```diff
@@ -0,0 +1,76 @@
+#----------------------
+## setup secure session
+#----------------------
+new-window -n turtlebot
+send-keys 'ros2 launch turtlebot3_gazebo turtlebot3_world.launch.py' Enter
+
+split-window -h
+send-keys 'ros2 run turtlebot3_teleop teleop_keyboard' Enter
+
+split-window -v
+send-keys 'ros2 topic echo /cmd_vel' Enter
+
+# Select teleop pane
+select-pane -t 1
+
+#----------------------
+# create navigation window
+#----------------------
+new-window -n navigation
+send-keys 'ros2 launch turtlebot3_navigation2 navigation2.launch.py use_sim_time:=true map:=maps/map.yaml' Enter
+
+split-window -h
+send-keys 'configs/initial_pose.sh'
+
+split-window -v
+send-keys 'configs/navigate_to_pose.sh'
+
+# Select initial_pose pane
+select-pane -t 1
+
+#----------------------
+# create mapping window
+#----------------------
+new-window -n mapping
+send-keys 'ros2 launch turtlebot3_cartographer cartographer.launch.py use_sim_time:=true'
+
+split-window -h
+send-keys 'ros2 run nav2_map_server map_saver -f maps/map'
+
+split-window -v
+send-keys 'ros2 topic info /map'
+
+# Select cartographer pane 
+select-pane -t 0
+
+#----------------------
+# create sros window
+#----------------------
+new-window -n sros
+send-keys 'tree keystore -d' Enter
+
+split-window -h
+send-keys 'ros2 security generate_artifacts -k keystore -p policies/tb3_gazebo_policy.xml'
+
+split-window -v
+send-keys 'env | grep ROS' Enter
+
+#----------------------
+# create reconnaissance window
```

Comment:
If you don't need run any new commands on startup, perhaps you could just add this window to the main common.conf one directory up instead of duplicating the entire config. I makes it a little harder to keep changes in sync.

Reply:
Noted. I didn't wanted to influence any other demos but knowing this is preferred, let me do it!

Reply:
Yea, I'd like to refactor AWS's PR to be included the common.config, or simple source it like so to add windows without altering the common config like so:

https://github.com/ros-swg/turtlebot3_demo/blob/424b79efb1baa217700d21b0ae6fdabff99abdad/configs/unsecure.conf#L9

You could do something like

``` bash
source $BYOBU_PREFIX/share/byobu/profiles/tmux

# setup reconnaissance session
new-session -s reconnaissance -n diagnostic -d
send-keys 'glances' Enter

# create reconnaissance window
new-window -n reconnaissance
send-keys 'aztarna -t ros2'

split-window -v
send-keys 'aztarna -t ros2 -d 0'

split-window -h
send-keys 'tshark -i eth0 -f "udp" -Y "rtps"'

# Select ROS2 reconnaissance pane
select-pane -t 0

source ../configs/common.conf
```

Reply:
All right, gave it a try @ruffsl. Removed the reconnaissance-specific config files and integrated the simple conf for reconnaissance into `common.conf`.